tests/brevitas/graph/equalization_fixtures.py

```diff
@@ -35,7 +35,7 @@
     'resnet18': [0.487, 0.952],
     'googlenet': [0.495, 0.982],
     'inception_v3': [0.497, 0.989],
-    'alexnet': [0.875, 0.875],}
+    'alexnet': [0.75, 0.75],}
```

Comment:
Earlier we were counting also the regions that we would then skip during the actual equalization process.
Now we're counting only the effective equalized region, so we have one less